### ISSUE TITLE
Direction Going Forward?

### ISSUE THREAD

**Issue by ybur-yug** [dojo-toulouse/elixir-koans#24]
Howdy,

I found these linked in the Awesome Elixir repo. I saw its been a while since anything was committed, and had been working on a similar project in the past but have learned enough since I want to generally scrap it. I was going to start over and thought forking this may be a great point.

However, if you plan on keeping actively developing it I would love to hear your plans and possibly just end up contributing via PR's instead of forking myself and needing to maintain it.

Great work! Hope to hear back.

Cheers,
Bobby

**Comment by apieum** [comment]
Thanks, I've no plans to maintain this repo, maybe others in this group have.
Do like you want, someone will manifest itself if it disagree ;)

**Comment by iamvery** [comment]
I've hit several snags with this project, and your comment above seems to indicate that it's not really being maintained anymore. Hopefully for more good than not, I've decided to create another Elixir koans project https://github.com/iamvery/elixir-koans. What are you feelings towards this?

**Comment by ybur-yug** [comment]
@iamvery I'd love to contribute and think that sounds reasonable. I'll check it out.

**Comment by iamvery** [comment]
@ybur-yug thanks! I wasn't particularly pleased with the output of failures. So I changed how koans on run on that project. Near term, it'd be super neat to hone the experience down to something closer to Jim's ruby koans.

**Comment by ybur-yug** [comment]
@iamvery I agree, but I also think the nature of elixir can offer us some pretty interesting possibly approaches. I'd love to experiment a bit, too. Especially since this isn't exactly a project that needs to move at warp speed.

I will check out where everything is and that issue referenced tonight and make a fork and start experimenting.

**Comment by iamvery** [comment]
:100:
